public_html/assets/css/nf-core-auto.css

```diff
@@ -1 +1,3 @@
+@import url('nf-core-light.css');
```

Comment:
What do you think about leaving these generated files out of git? Presumably they will be overwritten every time by `docker-compose` and could cause unnecessary diffs on PRs, in addition to general bloat.

Reply:
With leaving them out of git, do you mean untrack them or ignore them? I might be wrong, but if we ignore them, the css files generated by github actions would not be copied over during the deploy. if we untrack them, they will always come up in `git status`. Or were you thinking about a different way?

Reply:
I was thinking about ignoring them - but yes you're right, this would require a change in how we deploy the website updates. But to be honest I'm increasingly keen on that approach anyway.